frontend/dispatcher.lua

```diff
@@ -289,6 +290,9 @@ function Dispatcher:init()
         for y=1,#base[i].options do
             local option = base[i].options[y]
             if settingsList[option.name] ~= nil then
+                if option.name ~= nil and option.values ~= nil then
+                    settingsList[option.name].configurable = {name = option.name, values = option.values}
```

Comment:
I haven't looked at all the actions that reflects in the bottom menu - but are there some that would not be handled? Or it just magically works for all of them?

Reply:
the action from the bottom menu are:
https://github.com/koreader/koreader/blob/e157395660108032ca02c2acb3275d6012af23e7/frontend/dispatcher.lua#L134-L155

and they all are reflected here. ConfigDialog/Creopts use the same name value as configurable, and the values key it wat to set, (as opposed to args)